### DISCUSSION TITLE
백엔드에서 이미지 파일 처리가 가능하다면

### DISCUSSION THREAD

**Discussion by ois0886** [hansung-pocs/blog-android#167] [Ideas]
인스타와 같은 느낌으로 갤러리 게시판을 만드는 것은 어떨까
갤러리 게시판은 비회원까지도 볼 수 있게끔 하는 것이다.

생각이 떠오른 이유 : 
안드로이드 공식문서를 보다가 Grid를 사용하여 구현한 것을 보았다.
https://developer.android.com/jetpack/compose/lists

**Comment by jja08111** [comment]
저도 생각했던 부분이네요. 전체 이미지 혹은 추억용 이미지만 모아서 날짜별로 볼 수 있으면 좋겠네요.
